local_vars/local_vars.cc

```diff
@@ -228,6 +228,14 @@ class LocalNameInserter {
             return tree;
         }
 
+        int i = scopeStack.size() - 1;
```

Comment:
Maybe add an explicit cast around the `scopeStack.size()` call? Otherwise, this code may be slightly confusing to read, because it's not necessarily intuitive whether the conversion to `int` happens before the subtraction (yes) or if it happens after the subtraction (in which case this would be a bug).

Reply:
I certainly hope we never encounter a file with 2^31 nested scopes in it (and I have a feeling something else breaks first). In any case, I've changed this to use `size_t` throughout, never converting to `int`.